test/core/test/snapshot.test.ts

```diff
@@ -112,3 +115,18 @@ test('renders inline mock snapshot', () => {
     }
   `)
 })
+
+const root = path.resolve(__dirname, './fixtures')
+function run(testNamePattern?: string) {
+  return runVitest({
+    root,
+    testNamePattern,
+  })
+}
+
+describe('snapshot should not be obsolete', () => {
```

Comment:
When I run this test without your changes, it gives the same result. Your tests should validate that the change that you made is working.

If you just run this command in main, you can see that there are already no obsolete warnings. Please, create a test that actually tests things.